mergin/test/test_client.py

```diff
@@ -329,7 +330,7 @@ def test_list_of_push_changes(mc):
     mp = MerginProject(project_dir)
 
     shutil.copy(mp.fpath('inserted_1_A.gpkg'), mp.fpath(f_updated))
-
+    mc._auth_session["expire"] = dateutil.parser.parse("2020-02-04 12:11:31.490222+00:00")
```

Comment:
this seems bit odd with hardcoded expiration time? what is purpose of that? 

if we want to simulate expiration what about set it to now() or something like that?

Reply:
also is there a way to test login failure after token expiration?